presidio-analyzer/logging.ini

```diff
@@ -0,0 +1,27 @@
+[loggers]
+keys=root,presidio-analyzer
+
+[handlers]
+keys=consoleHandler
+
+[formatters]
+keys=simpleFormatter
+
+[logger_root]
+level=INFO
+handlers=consoleHandler
+
+[logger_presidio-analyzer]
+level=INFO
+handlers=consoleHandler
+qualname=presidio-analyzer
+propagate=0
+
+[handler_consoleHandler]
+class=StreamHandler
+level=INFO
+formatter=simpleFormatter
+args=(sys.stdout,)
+
+[formatter_simpleFormatter]
+format=%(asctime)s - %(name)s - %(levelname)s - %(message)s
```

Comment:
Needs checking that this is parsed correctly in app service logs

Reply:
Do we know if it does? We can update this later if we see it is not parsed correctly